services/self-service/src/main/java/com/epam/datalab/backendapi/service/impl/InfrastructureInfoServiceImpl.java

```diff
@@ -149,11 +149,13 @@ public InfrastructureMetaInfoDTO getInfrastructureMetaInfo() {
     @Override
     public void updateInfrastructureStatuses(UserInfo user, String endpoint, List<EnvResource> hostInstances, List<EnvResource> clusterInstances) {
         EnvResourceList envResourceList = EnvResourceList.builder()
-                        .hostList(hostInstances)
-                        .clusterList(clusterInstances)
-                        .build();
+                .hostList(hostInstances)
+                .clusterList(clusterInstances)
+                .build();
 
         EndpointDTO endpointDTO = endpointService.get(endpoint);
+        log.info("SELF-SERVICE: Send request to provisioning service:\n POST:{}, with EnvResources: {}", INFRASTRUCTURE_STATUS,
```

Comment:
Can we remove **SELF-SERVICE** from the log? It is already in the **selfservice.log** file.

Reply:
Ty, done.